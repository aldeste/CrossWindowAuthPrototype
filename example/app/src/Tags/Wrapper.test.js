import Wrapper from "./Wrapper";
import renderer from "react-test-renderer";
import React from "react";
// BUG: Theres a bug with jest-styled-components and styled-components 2.0.0-17, fix when updated.
// import { matcher, serializer } from "jest-styled-components";
//
// expect.addSnapshotSerializer(serializer);
// expect.extend(matcher);

// Once fixed, replace toMatchSnapshot() with toMatchStyledComponentsSnapshot()

describe("Wrapper tag", () => {
  it("Should be defined", () => {
    expect(Wrapper).toBeDefined();
  });

  it("Is a styled component", () => {
    expect(Wrapper.name).toBe("StyledComponent");
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<Wrapper />).toJSON();
    tree.props.className = "";
    expect(tree).toMatchSnapshot();
  });
});
