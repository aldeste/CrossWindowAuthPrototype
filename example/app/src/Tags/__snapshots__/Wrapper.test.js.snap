// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper tag Matches earlier configuration 1`] = `
.bnYeJt {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  border: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  min-height: : 0;
  min-width: 0;
}

.bnYeJt > * + * {
  margin-top: 0.75em;
}

.esaSay {
  color: #333;
  background: #fff;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
  font-size: 17px;
  font-weight: 400;
  letter-spacing: -0.021em;
  line-height: 1.52947;
  padding: 2rem 5rem;
}

.esaSay > * + * {
  margin-top: 2rem;
}

<div
  className="Wrapper__Wrapper-cZxMuO esaSay View__View-jelnfZ bnYeJt"
/>
`;
